.tutone.yml

```diff
@@ -587,13 +587,11 @@ packages:
       - name: EpochSeconds
         create_as: serialization.EpochTime
       - name: Minutes
-        create_as: string
+        create_as: int
       - name: NaiveDateTime
         create_as: string
       - name: Seconds
-        create_as: string
-      - name: Minutes
-        create_as: int
+        create_as: float64
```

Comment:
Should probably find a test that verifies `Seconds` comes back as a float, since it's a Scalar in GQL and could be anything...

Reply:
I agree.  Some return `0` which I want to confirm is a valid float.